### PR TITLE
Admin updates

### PR DESCRIPTION
**Pull request by smilerz**
updates to various admin pages

### CODE REVIEW

File: cookbook/admin.py [hunk 1]
```diff
@@ -163,10 +168,18 @@ def delete_unattached_steps(modeladmin, request, queryset):
 
 
 class StepAdmin(admin.ModelAdmin):
-    list_display = ('name', 'order',)
-    search_fields = ('name',)
+    list_display = ('recipe_and_name', 'order', 'space')
+    ordering = ('recipe__name', 'name', 'space', )
+    search_fields = ('name', 'recipe__name')
     actions = [delete_unattached_steps]
```
Comment (vabene1111) [inline cookbook/admin.py:174]: usually admin actions affect only selected entries, with this we will delete everything from every space, do you think that is sufficently clear to the user?

I am leaning towards yes as playing in admin is always only for advanced users but wanted to check

**Review by vabene1111 — COMMENTED** [review]
just one question/comment

**Comment by smilerz** [comment]
That was my thought process as well.  I can investigate to see if there is a way to throw a confirmation.

**Comment by vabene1111** [comment]
I think its fine, just wanted to double check, might change the text to "Delete ALL .." or something but other than that its fine, will get this merged (if I dont forget) for the next update.

**Review by vabene1111 — APPROVED** [review]

**Pull request merged**